runtime/doc/options.txt

```diff
@@ -7704,15 +7704,13 @@ A jump table for the options with a short description can be found at |Q_op|.
 						*'showtabpanel'* *'stpl'*
 'showtabpanel' 'stpl'	number	(default 0)
 			global
-			{not in Vi}
 			{not available when compiled without the |+tabpanel|
 			feature}
 	The value of this option specifies when the |tabpanel| with tab page
 	labels will be displayed:
 		0: never
 		1: only if there are at least two tab pages
 		2: always
-	This is for the non-GUI implementation of the tabpanel only.
```

Comment:
This should be kept, no?

Reply:
The 'showtabpanel' option controls the display of the tabpanel in both the CLI and GUI, so I believe this is unnecessary.

Reply:
yeah, makes sense. Thanks